File: Android_Studio/Java/Fragments/app/src/main/java/com/example/fragments/activity/MainActivity.java
package com.example.fragments.activity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.fragments.R;
import com.example.fragments.fragment.ContatosFragment;
import com.example.fragments.fragment.ConversasFragment;

public class MainActivity extends AppCompatActivity
{
    private Button botaoConversa, botaoContato;
    private ConversasFragment conversasFragment;
    private ContatosFragment contatosFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        botaoConversa = findViewById(R.id.botaoConversa);
        botaoContato = findViewById(R.id.botaoContato);

        // Remover sombra do ActionBar
        getSupportActionBar().setElevation(0);

        conversasFragment = new ConversasFragment();


        // Configurar objeto para o Fragmento
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        transaction.add(R.id.frameConteudo, conversasFragment);
        transaction.commit();

        botaoContato.setOnClickListener(new View.OnClickListener()
        {
            @Override
            public void onClick(View view)
            {
                contatosFragment = new ContatosFragment();

                FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
                transaction.replace(R.id.frameConteudo, contatosFragment);
                transaction.commit();
            }
        });

        botaoConversa.setOnClickListener((new View.OnClickListener()
        {
            @Override
            public void onClick(View view)
            {
                conversasFragment = new ConversasFragment();

                FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
                transaction.replace(R.id.frameConteudo, conversasFragment);
                transaction.commit();
            }
        }));
    }
}

File: Android_Studio/Kotlin/Passando_Dados_Activity/app/src/main/java/com/example/passandodadosactivities/MainActivity.kt
package com.example.passandodadosactivities

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity()
{

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button_passar_dados.setOnClickListener {
            val intent = Intent(applicationContext, SegundaActivity:: class.java)

            // Passar dados
            intent.putExtra("nome", "Giovani")
            intent.putExtra("idade", 27)
            startActivity(intent)
        }
    }

}
